### PR TITLE
DOCS-2522: Update changelog

### PR DESCRIPTION
**Pull request by JessamyT**

### CODE REVIEW

File: docs/appendix/changelog.md [hunk 1]
```diff
@@ -19,10 +19,38 @@ outputs:
...
+The motion service API parameter `GeoObstacle` has been renamed to `GeoGeometry`.
+This affects users of the [`MoveOnGlobe()`](/services/motion/#moveonglobe) method.
+
+{{% /changelog %}}
+
+{{< changelog date="2024-05-09" color="changed" title="Return type of GetImage" >}}
 
 The Python SDK introduced a new image container class called [`ViamImage`](https://python.viam.dev/autoapi/viam/components/camera/index.html#viam.components.camera.ViamImage).
-The camera API's [`GetImage()`](/components/camera/#getimage) API method now returns a `ViamImage` type, and the vision service's [GetDetections()](/services/vision/#getdetections) and [GetClassifications()](/services/vision/#getclassifications) take in `ViamImage` as a parameter.
+The camera API's [`GetImage()`](/components/camera/#getimage) API method now returns a `ViamImage` type, and the vision service's [`GetDetections()`](/services/vision/#getdetections) and [`GetClassifications()`](/services/vision/#getclassifications) take in `ViamImage` as a parameter.
```
Comment (andf-viam) [inline docs/appendix/changelog.md:53]: Nit: phrasing, explicitness:

```suggestion
The camera component's [`GetImage()`](/components/camera/#getimage) API method now returns a `ViamImage` type, and the vision service's [`GetDetections()`](/services/vision/#getdetections) and [`GetClassifications()`](/services/vision/#getclassifications) API methods now take a `ViamImage` as a parameter.
```

**Review by andf-viam — APPROVED** [review]
LGTM, with one optional nit. Thank you for including commented sections -- I will uncomment once I've merged in the corresponding changes - appreciated!

File: docs/appendix/changelog.md (hunk 1 above)
Comment (JessamyT) [reply docs/appendix/changelog.md:53]: oops I didn't mean to say API twice! thanks for catching

Comment (JessamyT) [reply docs/appendix/changelog.md:53]:
```suggestion
The camera component's [`GetImage()`](/components/camera/#getimage) method now returns a `ViamImage` type, and the vision service's [`GetDetections()`](/services/vision/#getdetections) and [`GetClassifications()`](/services/vision/#getclassifications) methods take in `ViamImage` as a parameter.
```

**Pull request merged**